client/components/suggestions/index.jsx

```diff
@@ -227,10 +254,20 @@ const Suggestions = React.createClass( {
 				<div className="suggestions__category" key={ key }>
 					<span className="suggestions__category-name">{ key }</span>
 					<span className="suggestions__category-counter">
-						{ this.translate( '%(filtered)s of %(total)s', {
+						{ i18n.translate( '%(filtered)s of %(total)s', {
 							args: { filtered, total }
 						} ) }
 					</span>
+					{ this.props.terms[ key ].length > suggestions[ key ].length &&
+						<ShowAll
+							onClick={ this.onShowAllClick }
+							cat={ key }
+							txt={ i18n.translate( 'Show All' ) } /> }
+					{ key === this.state.showAll &&
+						<ShowAll
```

Comment:
Can we use a better set of labels/names here?

* `ShowAll` → `SuggestionsButtonAll` or something you see fit
* `props.txt` → `props.label`
* `props.cat` → I'm not exactly sure what value this holds... showAll? key?